src/modules/quoiFeur/quoiFeur.module.ts

```diff
@@ -0,0 +1,45 @@
+import { SlashCommandBuilder } from 'discord.js';
+
+import { config } from '../../config';
+import type { BotModule } from '../../types/bot';
+import {
+  addQuoiFeurToChannel,
+  deleteRoleMutedByBot,
+  reactOnEndWithQuoi,
+  removeQuoiFeurFromChannel,
+} from './quoiFeur.helpers';
+
+export const quoiFeur: BotModule = {
+  slashCommands: [
+    {
+      schema: new SlashCommandBuilder()
+        .setName('quoi-feur')
+        .setDescription('Manage quoi-feur game in the channel')
+        .addSubcommand((subcommand) =>
+          subcommand.setName('add').setDescription('Add the quoi-feur game to the channel'),
+        )
+        .addSubcommand((subcommand) =>
+          subcommand.setName('remove').setDescription('Remove the quoi-feur game from the channel'),
+        )
+        .toJSON(),
+      handler: {
+        add: async (interaction) => {
```

Comment:
You don't need to wrap this inside an arrow-function, you can pass directly `addQuoiFeurToChannel` to `add`.

Whenever a command throw/crash, it already prints out to the console as an error, so the .catch is unnecessary.

Same with `remove` below.